internal/db/start/start.go

```diff
@@ -81,13 +80,18 @@ func NewContainerConfig() container.Config {
 			Timeout:  2 * time.Second,
 			Retries:  3,
 		},
-		Entrypoint: []string{"sh", "-c", `cat <<'EOF' > /etc/postgresql.schema.sql && cat <<'EOF' > /etc/postgresql-custom/pgsodium_root.key && docker-entrypoint.sh postgres -D /etc/postgresql
+		Entrypoint: []string{"sh", "-c", `
+cat <<'EOF' > /etc/postgresql.schema.sql && \
+cat <<'EOF' > /etc/postgresql-custom/pgsodium_root.key && \
+cat <<'EOF' >> /etc/postgresql/postgresql.conf && \
```

Comment:
Isn't it problematic to append to `postgresql.conf` on every db restart? the file will keep growing in size. I'm not sure what would be a better solution though

Reply:
I don't think `/etc/postgresql` is mounted in a volume, so restart should always start with the same file. I'll have a look to be sure.

Reply:
thanks for verifying